client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -1,22 +1,107 @@
+import { JetpackLogo } from '@automattic/components/src/logos/jetpack-logo';
 import { useQuery } from '@tanstack/react-query';
-import { __experimentalVStack as VStack } from '@wordpress/components';
+import {
+	__experimentalGrid as Grid,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Icon,
+	Tooltip,
+} from '@wordpress/components';
 import { cloneElement } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
-import { wordpress } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	chartBar,
+	cloud,
+	megaphone,
+	next,
+	removeBug,
+	shield,
+	search,
+	video,
+	wordpress,
+} from '@wordpress/icons';
 import filesize from 'filesize';
 import { siteMediaStorageQuery } from '../../app/queries/site-media-storage';
 import { siteMetricsQuery } from '../../app/queries/site-metrics';
 import { siteCurrentPlanQuery } from '../../app/queries/site-plans';
 import { sitePurchaseQuery } from '../../app/queries/site-purchases';
 import { Stat } from '../../components/stat';
 import { DotcomPlans } from '../../data/constants';
+import { hasPlanFeature } from '../../utils/site-features';
+import { isSelfHostedJetpackConnected } from '../../utils/site-types';
 import { getSiteDisplayUrl } from '../../utils/site-url';
 import OverviewCard from '../overview-card';
+import type { JetpackModules } from '../../data/constants';
 import type { Site, Plan, Purchase } from '../../data/types';
 
+import './style.scss';
+
 const MINIMUM_DISPLAYED_USAGE = 2.5;
 const ALERT_PERCENT = 80;
 
+const JETPACK_MODULES = [
+	{
+		// See: https://github.com/Automattic/jetpack/blob/46e8a181299c296ac8e108daa2f2c6f9761a2d82/projects/packages/my-jetpack/src/products/class-stats.php#L82
+		id: 'stats-paid',
```

Comment:
The `JetpackFeatures` enum has `stats` instead. Is this expected?

Also I see that the new `JetpackFeatures` enum is not actually being used. Should we use it for these `id` fields? Like `id: JetpackFeatures.CLOUD_CRITICAL_CSS`

Reply:
Yup, refactored to use the constants 🙂